src/scipp/core/data_group.py

```diff
@@ -57,6 +58,9 @@ def __init__(self, /, *args, **kwargs):
         if not all([isinstance(k, str) for k in self._items.keys()]):
             raise ValueError("DataGroup keys must be strings.")
 
+    def __copy__(self) -> DataGroup:
+        return DataGroup(copy.copy(self._items))
+
```

Comment:
What about `__deepcopy__`? Does it do the right thing by default?

Reply:
Yes, see tests.